README.md

```diff
@@ -31,6 +31,26 @@ Currently, the output shouldn't be always trusted and isn't perfect but should b
 - feel free to open issue or discuss! I don't have github notifications so also feel free to go ping me on the typst discord server (at Astrale).
 - once discussed, you may open a PR, not before cause I'm a bit chaotic and this is wip so things change fast and I would hate it if you lost your time.
 
+# Pre-commit hook
+
+Refer <https://pre-commit.com>, add `.pre-commit-config.yaml`:
+
+```yaml
+repos:
+  - repo: https://github.com/astrale-sharp/typst-fmt
+    rev: "the latest git tag"
+    hooks:
+      - id: typst-fmt
```

Comment:
testing this only works if i change this to typstfmt

Reply:
Fixed.